autopush/tests/test_rs_integration.py

```diff
@@ -45,12 +51,38 @@
 MESSAGE_TABLE = os.environ.get("MESSAGE_TABLE", "message_int_test")
 
 
+def get_free_port():
+    s = socket.socket(socket.AF_INET, type=socket.SOCK_STREAM)
+    s.bind(('localhost', 0))
+    address, port = s.getsockname()
+    s.close()
+    return port
+
+
 def setup_module():
     logging.getLogger('boto').setLevel(logging.CRITICAL)
     if "SKIP_INTEGRATION" in os.environ:  # pragma: nocover
         raise SkipTest("Skipping integration tests")
 
 
+class MockMegaphoneRequestHandler(BaseHTTPRequestHandler):
+    API_PATTERN = re.compile(r'/v1/broadcasts')
+    services = {}
+    polled = Event()
+
+    def do_GET(self):
+        if re.search(self.API_PATTERN, self.path):
+            self.send_response(requests.codes.ok)
+            self.send_header('Content-Type', 'application/json; charset=utf-8')
+            self.end_headers()
+            response_content = json.dumps(
+                {"broadcasts": MockMegaphoneRequestHandler.services}
```

Comment:
not `self.services`?

Reply:
I was having a difficult time remembering if that would get a new local instance copy vs. the class copy, self.services should work since it is never assigned to in the instance I think?